src/main/kotlin/org/jitsi/nlj/Transceiver.kt

```diff
@@ -131,7 +131,7 @@ class Transceiver(
      * this transceiver is associated with) to be processed by the receiver pipeline.
      */
     fun handleIncomingPacket(p: PacketInfo) {
-        packetIOActivity.lastRtpPacketReceivedTimestamp = clock.instant()
+        packetIOActivity.lastRtpPacketReceived = clock.instant()
```

Comment:
It feels weird to me to not have at least some notion of 'time' in the name of these.

Reply:
I think it's pretty clear from the type being `Instant` (besides, they have docs now)

Reply:
Yeah but the type isn't always there when code is being read.